modules/top-level/nixpkgs.nix

```diff
@@ -55,31 +55,71 @@ in
         > Use this option with care.
       '';
     };
+
+    overlays = lib.mkOption {
+      type =
+        let
+          overlayType = lib.mkOptionType {
+            name = "nixpkgs-overlay";
+            description = "nixpkgs overlay";
+            check = lib.isFunction;
+            merge = lib.mergeOneOption;
+          };
+        in
+        lib.types.listOf overlayType;
+      default = [ ];
+      # FIXME: use an example that is topical for vim
+      example = lib.literalExpression ''
+        [
+          (self: super: {
+            openssh = super.openssh.override {
+              hpnSupport = true;
+              kerberos = self.libkrb5;
+            };
+          })
+        ]
```

Comment:
Any ideas for a better vim-related example?

Reply:
Was wondering if we could do one that adds a tree-sitter-X grammar to the treesitter package ?

Reply:
overlays = [ inputs.neovim-nightly-overlay.overlays.default ];
?